plugins/module_utils/netbox_utils.py

```diff
@@ -1311,7 +1325,7 @@ def check_mutually_exclusive(self, terms, module_parameters):
         return results
 
     def _check_required_if(self, spec, param=None):
-        """ ensure that parameters which conditionally required are present """
+        """ensure that parameters which conditionally required are present"""
```

Comment:
this change appears to be unrelated to the subject of the PR.

Reply:
Yeah, not sure why black suddenly decided to reformat that doc string.